CMakeLists.txt

```
@@ -122,7 +122,7 @@ add_definitions(${DAWN_EXTERNAL_DEFINITIONS})
 yoda_report_result("Package summary" ${package_info})
 
 # Add clang-format target
-yoda_add_target_clang_format(DIRECTORIES ${CMAKE_SOURCE_DIR}/src EXTENSION ".h;.cpp")
+yoda_add_target_clang_format(DIRECTORIES ${CMAKE_SOURCE_DIR}/src ${CMAKE_SOURCE_DIR}/test EXTENSION ".h;.cpp")
```

```suggestion
yoda_add_target_clang_format(DIRECTORIES ${CMAKE_PROJECT_DIR}/src ${CMAKE_PROJECT_DIR}/test EXTENSION ".h;.cpp")
```
I know, not your fault, but let's just fix it as we go